Definitions.md

```diff
@@ -0,0 +1,16 @@
+#Definition Of Terms
+
+##Job
```

Comment:
To provide content for that diagram idea, I'm thinking:
Job/TaskInstance is an instantiation of a Task. 
Workflow is composed of Tasks. 
WorkflowInstance is an instantiation of a Workflow. 
WorkflowInstance is composed of Jobs/TaskInstances. 

Not sure how Tool fits in yet.